Tests/Functional/Asynchronous/Expectations/main.swift

```diff
@@ -84,11 +134,16 @@ class ExpectationsTestCase: XCTestCase {
             ("test_waitingForAnExpectationFulfilledAfterTheTimeout_fails", test_waitingForAnExpectationFulfilledAfterTheTimeout_fails),
             ("test_whenTimeoutIsImmediate_andAllExpectationsAreFulfilled_passes", test_whenTimeoutIsImmediate_andAllExpectationsAreFulfilled_passes),
             ("test_whenTimeoutIsImmediate_butNotAllExpectationsAreFulfilled_fails", test_whenTimeoutIsImmediate_butNotAllExpectationsAreFulfilled_fails),
+            ("test_observeNotificationWithName_passes", test_observeNotificationWithName_passes),
+            ("test_observeNotificationWithNameAndObject_passes", test_observeNotificationWithNameAndObject_passes),
+            ("test_observeNotificationWithNameAndObject_butExpectingNoObject_passes", test_observeNotificationWithNameAndObject_butExpectingNoObject_passes),
+            ("test_observeNotificationWithIncorrectName_fails", test_observeNotificationWithIncorrectName_fails),
+            ("test_observeNotificationWithIncorrectObject_fails", test_observeNotificationWithIncorrectObject_fails),
```

Comment:
nit-pick: My personal preference would be to have these tests in a separate test file, `Tests/Functional/Asynchronous/NotificationExpectations/main.swift`. I'd like to keep test files as small as possible.